rhel/coalescer_test.go

```diff
@@ -2,340 +2,466 @@ package rhel
 
 import (
 	"context"
-	"strconv"
+	"fmt"
+	"net/url"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/quay/zlog"
 
 	"github.com/quay/claircore"
 	"github.com/quay/claircore/indexer"
 	"github.com/quay/claircore/test"
+	"github.com/quay/claircore/toolkit/types/cpe"
```

Comment:
not sure if this belongs here or the other section because this is from a different module